### PR TITLE
Use canonical names as values in Salesforce forms

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/pages/comp-copy/comp-copy.hbs [hunk 1]
```diff
@@ -63,18 +63,18 @@
             <span>Book Requested</span>
             <select name="00NU00000053nzR" required />
                 <option value="" disabled selected>Please Select</option>
-                <option value="Anatomy and Physiology">Anatomy and Physiology</option>
+                <option value="Anatomy & Physiology">Anatomy and Physiology</option>
```
Comment (dak) [inline src/app/pages/comp-copy/comp-copy.hbs]: Should this be `&amp;`?

**Comment by dak** [comment]
It seems like it might be useful to make this into a model/data that can be loaded dynamically by the template, rather than repeating it in multiple files.

File: src/app/pages/comp-copy/comp-copy.hbs (hunk 1 above)
Comment (RoyEJohnson) [inline src/app/pages/comp-copy/comp-copy.hbs]: It took me a while to find an answer, but it looks like yes.

**Comment by RoyEJohnson** [comment]
I actually have a file with the book data. The challenge is that not all the forms use all the titles.

**Pull request merged**